### DISCUSSION TITLE
Erro na impressão da O.S

### DISCUSSION THREAD

**Discussion by xavierfilho** [RamonSilva20/mapos#1555] [General]
ola, tudo bem?
Estou com problema na impresão de A4 na O.S ele dar erro no pix, alguem tem ideia o que seria

**Comment by xavierfilho** [comment]
$pix = (new StaticPayload())
            ->applyValidCharacters()
            ->applyUppercase()
            ->setPixKey(getPixKeyType($pixKey), $pixKey)
            ->setMerchantName($emitente->nome, true)
            ->setMerchantCity($emitente->cidade, true)
            ->setAmount($amount)
            ->setTid($id)
            ->setDescription(sprintf("%s OS %s", $emitente->nome, $id), true);
 
        return $pix->getQRCode();

**Comment by Pr3d4dor** [comment]
@xavierfilho Qual é o erro exato?

**Reply by xavierfilho** [reply]
> 
![Captura de Tela 2021-11-24 às 12 02 25](https://user-images.githubusercontent.com/67480581/143262977-5ae787e4-09e7-496d-80db-8628770e25db.png)

Boa Tarde, tudo bem? 
Envie uma foto com o erro, obrigado por responder.

**Reply by xavierfilho** [reply]
sempre que envio para impressao a4

**Comment by xavierfilho** [comment]
Ola, tudo bem?

Ela dar um erro de pix e estava normal antes da atualização.
[image: Captura de Tela 2021-11-24 às 12.02.25.png]

Em ter., 23 de nov. de 2021 às 20:12, Gianluca Bine <
***@***.***> escreveu:

**Comment by Pr3d4dor** [comment]
@xavierfilho Precisa rodar composer install
